Backend/app/dashboard/provider-dashboard/src/API/ProviderPlatform/DynamicOfferDriver/Subscription.hs

```diff
@@ -61,6 +62,10 @@ type CurrentPlan =
   ApiAuth 'DRIVER_OFFER_BPP 'SUBSCRIPTION 'CURRENT_PLAN
     :> SD.CurrentPlan
 
+type ResumePlan =
+  ApiAuth 'DRIVER_OFFER_BPP 'SUBSCRIPTION 'CURRENT_PLAN
```

Comment:
Entity should be `RESUME_PLAN`. Please create that entity

Reply:
Can you also write migrations to provide access to these entities?

Reply:
okay